packages/service-library/tests/test_redis.py

```diff
@@ -227,7 +227,11 @@ async def test_lock_acquired_in_parallel_to_update_same_resource(
     faker: Faker,
 ):
     INCREASE_OPERATIONS: Final[int] = 250
-    INCREASE_BY: Final[int] = 10
+    # FIXME: DK changes this value from 10-->8 Aug2024
+    # when introducing opentelemetry tracing since this test would
+    # always fail in the github-runners, but not on local machines...
+    # This may need some refactoring/investigation
+    INCREASE_BY: Final[int] = 8
```

Comment:
Ok we do not do FIXMEs or TODOs. They get forgotten and nobody ever will look at them.
From the blame it seems the test was done by ANE.

Please change the FIXME into a NOTE, create an issue in simcore with this, add whatever is necessary to understand it, assign it to ANE, and put the link in the note. Thanks!

Reply:
Indeed, I thought we discussed that we should create an issue for @GitHK such that you both can look into it together.